src/test/java/com/iexec/worker/config/BlockchainAdapterConfigurationServiceTest.java

```diff
@@ -13,6 +13,12 @@
 import static org.mockito.Mockito.when;
 
 public class BlockchainAdapterConfigurationServiceTest {
+    private static final Integer CHAIN_ID = 0;
+    private static final boolean IS_SIDECHAIN = true;
+    private static final String NODE_ADDRESS = "0x1";
```

Comment:
```suggestion
    private static final String NODE_ADDRESS = "https://node";
```
Just for your understanding

Reply:
Right, that's better. Thanks :)